jina/helper.py

```diff
@@ -1618,3 +1618,30 @@ def is_port_free(host, port):
             return False
         else:
             return True
+
+
+def _parse_port(port: str) -> Union[int, List]:
```

Comment:
```suggestion
def _parse_ports(port: str) -> Union[int, List]:
```
i think this should be plural

Reply:
will do